tools/setup_helpers/gen_version_header.py

```diff
@@ -0,0 +1,67 @@
+# Ideally, there would be a way in Bazel to parse version.txt
+# and use the version numbers from there as substitutions for
+# an expand_template action. Since there isn't, this silly script exists.
+
+import argparse
+
+def parse_version(version: str) -> (int, int, int):
+    """
+    Parses a version string into (major, minor, patch) version numbers.
+
+    Args:
+      version: Full version number string, possibly including revision / commit hash.
+
+    Returns:
+      An int 3-tuple of (major, minor, patch) version numbers.
+    """
+    # Extract version number part (i.e. toss any revision / hash parts).
+    version_number_str = version
+    for i in range(len(version)):
+        c = version[i]
+        if not (c.isdigit() or c == "."):
+            version_number_str = version[:i]
+            break
+
+    return tuple([int(n) for n in version_number_str.split(".")])
+
```

Comment:
[PEP-8](https://www.python.org/dev/peps/pep-0008/#blank-lines) suggest 2 blank lines between functions

Reply:
@jbschlosser fix this, and we are good to land